devices/src/lib.rs

```diff
@@ -43,12 +43,12 @@ pub enum EpollHandlerPayload {
     DrivePayload(File),
     /// Used to mutate current RateLimiter settings. The buckets are rx_bytes, rx_ops,
     /// tx_bytes, and tx_ops, respectively.
-    PatchRateLimiters(
-        Option<TokenBucket>,
-        Option<TokenBucket>,
-        Option<TokenBucket>,
-        Option<TokenBucket>,
-    ),
+    PatchRateLimitersPayload {
```

Comment:
Rename this to `PatchNetRateLimitersPayload` since it's net-specific.

Reply:
Done.